index.html

```diff
@@ -1734,39 +1783,114 @@ <h3>Document Encoding</h3>
+          <li>DAPT processors targeted at one version of the specification to process <a>DAPT documents</a>
+            authored for an earlier version with similar or identical behaviour to a DAPT processor
+            targeted at that earlier version.</li>
+        </ul>
+        <p>A <a>DAPT document</a> that conforms to a future version of the specification could specify
+          conformance to multiple DAPT <a>content profiles</a>.</p>
+        <section id="unrecognised-elements-and-attributes">
+          <h4>Unrecognised vocabulary</h4>
+          <p><dfn>Unrecognised vocabulary</dfn> is the set of elements and attributes that are
+            not associated with <a>features</a> that the processor supports.</p>
+          <p>A <a>transformation processor</a> MUST prune <a>unrecognised vocabulary</a> that is
+            neither an attribute nor a descendant of
+            a <code>&lt;metadata&gt;</code> element.</p>
+          <p>A <a>transformation processor</a> SHOULD preserve <a>unrecognised vocabulary</a> that is
+            either an attribute or a descendant of
+            a <code>&lt;metadata&gt;</code> element.</p>
+          <p class="note">See also <a href="#ttp-contentprofiles"></a> which prohibits the signalling of profile
+          conformance to profiles that the <a>transformation processor</a> does not support.</p>
+          <p>
+            After <a href="#using-computed-attribute-values">attribute value computation</a>,
+            a <a>presentation processor</a> SHOULD ignore elements that it does not recognise and support.
+          </p>
+          <p class="note">
+            The above constraint is specified as being after attribute value computation because it is possible
+            that an implementation recognises and supports attributes present only on particular elements,
+            for example those corresponding to the DAPT data model.
+            As described in <a href="#using-computed-attribute-values"></a> it is important that processor implementations
+            do not ignore such attributes when present on other elements.
+          </p>
+          <p>A <a>presentation processor</a> SHOULD ignore attributes that it does not recognise and support.</p>
+        </section>
+        <section>
+          <h4>Special considerations for foreign vocabulary</h4>
+          <p>
+            A <a>DAPT Document</a> MAY contain <a>foreign vocabulary</a> that is neither specifically permitted nor forbidden
+            by the profiles signalled in <a>ttp:contentProfiles</a>.
+          </p>
+          <p><dfn>Foreign vocabulary</dfn> is the subset of <a>unrecognised vocabulary</a> that consists of
+            those elements and attributes whose namespace
+            is not one of the namespaces listed in <a href="#namespaces"></a> and
+            those attributes in the global namespace that are not otherwise defined in DAPT or in [[TTML2]].</p>
```

Comment:
Can you swap the order of these 2 sentences?

Reply:
Done in c68a08a.